### PR TITLE
ANDROID-10672 New corner radius

### PR DESCRIPTION
**Pull request by yamal-alm**
### :goal_net: What's the goal?
As part of Mistica Evolution, there have been some changes related to corner radius and other small changes

### :construction: How do we do it?
Change (compose and xml) the corner radius here and there from 4px to 8px (actually dp on Android) in the following components:
- Cards (data and media)
- Callout
- Popover
- Boxed list items
- Empty state card
- Forms

Other fixes:
- Change feedback subtitle style from light to regular
- Empty state card icon size from 64 to 48

### ☑️ Checks
- [x] I updated the documentation (readmes, wikis, etc). If no need to update documentation explain why.
- [x] Tested with dark mode.
- [x] Tested with API 21.

### :test_tube: How can I test this?
- [x] 🖼️ Screenshots/Videos
- [x] [Mistica App download link](https://install.appcenter.ms/orgs/tuenti-organization/apps/mistica-catalog/distribution_groups/public)
Captura de pantalla 2022-06-21 a las 11 41 02
- [x] Reviewed by Mistica design team

### XML
| component | before | after |
| ---- | ---- | ---- |
| media card | Captura de pantalla 2022-06-21 a las 10 54 06| Captura de pantalla 2022-06-21 a las 10 57 34|
| data card | Captura de pantalla 2022-06-21 a las 10 53 46| Captura de pantalla 2022-06-21 a las 10 45 59|
| callout | Captura de pantalla 2022-06-21 a las 10 53 25|Captura de pantalla 2022-06-21 a las 10 45 33|
| popover | Captura de pantalla 2022-06-21 a las 10 54 35| Captura de pantalla 2022-06-21 a las 10 48 26|
| boxed list item | Captura de pantalla 2022-06-21 a las 10 55 01| Captura de pantalla 2022-06-21 a las 10 47 47|
| empty state card | Captura de pantalla 2022-06-21 a las 10 55 45| Captura de pantalla 2022-06-21 a las 10 46 34|
| forms | Captura de pantalla 2022-06-21 a las 10 55 26| Captura de pantalla 2022-06-21 a las 10 47 03|
| feedback subtitle | Captura de pantalla 2022-06-21 a las 11 07 44| Captura de pantalla 2022-06-21 a las 10 59 13|
| empty state card icon | Captura de pantalla 2022-06-21 a las 11 16 59| Captura de pantalla 2022-06-21 a las 11 19 56|

### Compose (callout and popover still don't have a Compose version)
| component | before | after |
| ---- | ---- | ---- |
| media card | Captura de pantalla 2022-06-21 a las 10 51 49| Captura de pantalla 2022-06-21 a las 10 41 12|
| data card | Captura de pantalla 2022-06-21 a las 10 51 28| Captura de pantalla 2022-06-21 a las 10 40 20|
| boxed list item | Captura de pantalla 2022-06-21 a las 10 52 16| Captura de pantalla 2022-06-21 a las 10 41 58|
| empty state card | Captura de pantalla 2022-06-21 a las 10 52 36| Captura de pantalla 2022-06-21 a las 10 44 30|
| forms | Captura de pantalla 2022-06-21 a las 10 50 51| Captura de pantalla 2022-06-21 a las 10 39 19|
| feedback subtitle | Captura de pantalla 2022-06-21 a las 11 06 40| Captura de pantalla 2022-06-21 a las 11 01 49|
| empty state card icon | Captura de pantalla 2022-06-21 a las 11 17 55| Captura de pantalla 2022-06-21 a las 11 19 21|

### CODE REVIEW

File: library/src/main/res/drawable/bg_textinputedittext.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="rectangle">
     <solid android:color="?colorBackgroundContainer" />
-    <corners android:radius="5dp" />
+    <corners android:radius="8dp" />
```
Comment (manolovn) [inline library/src/main/res/drawable/bg_textinputedittext.xml:4]: should we extract this dimension to a dimen resource?

**Review by manolovn — APPROVED** [review]
AWESOME description, thanks you very much!

File: library/src/main/res/drawable/bg_textinputedittext.xml (hunk 1 above)
Comment (yamal-alm) [reply library/src/main/res/drawable/bg_textinputedittext.xml:4]: I think we are using separated dimens files where the dimen is used in multiple places

**Review by DevPabloGarcia — APPROVED** [review]
Nicee! 😄

**Comment by AnaMontes11** [comment]
I keep seeing the Empty state card (XML and Compose) with 4px corner radius @yamal-alm

**Review by AnaMontes11 — APPROVED** [review]
Nice work @yamal-alm ! 😍

**Pull request merged**